x-pack/plugins/ml/public/alerting/register_ml_alerts.ts

```diff
@@ -0,0 +1,93 @@
+
+export function registerMlAlerts(
+  alertTypeRegistry: MlStartDependencies['triggersActionsUi']['alertTypeRegistry']
+) {
+  alertTypeRegistry.register({
+    id: ML_ALERT_TYPES.ANOMALY_DETECTION,
+    description: i18n.translate('xpack.ml.alertTypes.anomalyDetection.description', {
+      defaultMessage: 'Alert when anomaly detection jobs results match the condition.',
+    }),
+    iconClass: 'bell',
+    documentationUrl(docLinks) {
+      return `${docLinks.ELASTIC_WEBSITE_URL}guide/en/machine-learning/${docLinks.DOC_LINK_VERSION}/ml-configuring-alerts.html`;
+    },
+    alertParamsExpression: lazy(() => import('./ml_anomaly_alert_trigger')),
+    validate: (alertParams: MlAnomalyDetectionAlertParams) => {
+      const validationResult = {
+        errors: {
+          jobSelection: new Array<string>(),
+          severity: new Array<string>(),
+          resultType: new Array<string>(),
+        },
+      };
+
+      if (
+        !alertParams.jobSelection?.jobIds?.length &&
+        !alertParams.jobSelection?.groupIds?.length
+      ) {
+        validationResult.errors.jobSelection.push(
+          i18n.translate('xpack.ml.alertTypes.anomalyDetection.jobSelection.errorMessage', {
+            defaultMessage: 'Job selection is required',
+          })
+        );
+      }
+
+      if (alertParams.severity === undefined) {
+        validationResult.errors.severity.push(
+          i18n.translate('xpack.ml.alertTypes.anomalyDetection.severity.errorMessage', {
+            defaultMessage: 'Anomaly severity is required',
+          })
+        );
+      }
+
+      if (alertParams.resultType === undefined) {
+        validationResult.errors.resultType.push(
+          i18n.translate('xpack.ml.alertTypes.anomalyDetection.resultType.errorMessage', {
+            defaultMessage: 'Result type is required',
+          })
+        );
+      }
+
+      return validationResult;
+    },
+    requiresAppContext: false,
+    defaultActionMessage: i18n.translate(
+      'xpack.ml.alertTypes.anomalyDetection.defaultActionMessage',
+      {
+        defaultMessage: `Elastic Stack Machine Learning Alert:
+- Job IDs: \\{\\{#context.jobIds\\}\\}\\{\\{context.jobIds\\}\\} - \\{\\{/context.jobIds\\}\\}
+- Time: \\{\\{context.timestampIso8601\\}\\}
+- Anomaly score: \\{\\{context.score\\}\\}
+
+Alerts are raised based on real-time scores. Remember that scores may be adjusted over time as data continues to be analyzed.
+
+\\{\\{! Section might be not relevant if selected jobs don't contain influencer configuration \\}\\}
+Top influencers:
+\\{\\{#context.topInfluencers\\}\\}
+  \\{\\{influencer_field_name\\}\\} = \\{\\{influencer_field_value\\}\\} [\\{\\{score\\}\\}]
+\\{\\{/context.topInfluencers\\}\\}
+
+Top records:
+\\{\\{#context.topRecords\\}\\}
+  \\{\\{function\\}\\}(\\{\\{field_name\\}\\}) \\{\\{by_field_value\\}\\} \\{\\{over_field_value\\}\\} \\{\\{partition_field_value\\}\\} [\\{\\{score\\}\\}]
```

Comment:
We need to floor the scores displayed in the preview and the alert, to match what we do in the Anomaly Explorer, including `< 1` if possible for scores less than 1. Can you use `getFormattedSeverityScore` from `anomaly_utils.ts`?